.bumpversion.cfg

```diff
@@ -0,0 +1,8 @@
+[bumpversion]
+current_version = 1.0.0
+commit = True
+tag = True
+
```

Comment:
We prefer not creating tags when bumping a new version, as this operation is performed at the very beginning of a new development sprint, just after previous version is tagged and released from master branch. So please set ``tag`` to False.